.changeset/config.json

```
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://unpkg.com/@changesets/config/schema.json",
+	"changelog": "@changesets/changelog-github",
```

```suggestion
	"changelog": [
		"@changesets/changelog-github",
		{ "repo": "db-ux-design-system/core-web" }
	],
```

https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#changelog-false-or-a-path mentions this `repo` declaration and https://github.com/changesets/changesets/blob/main/.changeset/config.json even also includes it.